packages/fxa-settings/src/components/SecurityPanel/index.test.tsx

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import SecurityPanel from '.';
+import { assert } from 'console';
+
+describe('SecurityPanel', () => {
+  it('renders SecurityPanel with expected attributes', () => {
+    render(<SecurityPanel />);
+
+    // auto fail this one so I remember to come back and fix it.
+    // - Do these panels need a 'role'?
+    expect(screen.getByTestId('settings-security')).toBeCalled;
```

Comment:
@LZoog Mentioned this in the jira ticket, but wondering if you think it makes sense to set a `role` on these panels?

Reply:
As mentioned in Slack I don't believe we need that here. We have `role="img" for inlined SVGs but this should be read properly/by our HTML semantics (a new section/divs).